common/compiler/helpers.go

```diff
@@ -27,9 +27,10 @@ var versionRegexp = regexp.MustCompile(`([0-9]+)\.([0-9]+)\.([0-9]+)`)
 
 // Contract contains information about a compiled contract, alongside its code and runtime code.
 type Contract struct {
-	Code        string       `json:"code"`
-	RuntimeCode string       `json:"runtime-code"`
-	Info        ContractInfo `json:"info"`
+	Code        string            `json:"code"`
+	RuntimeCode string            `json:"runtime-code"`
+	Info        ContractInfo      `json:"info"`
+	Hashes      map[string]string `json:"hashes"`
```

Comment:
Is there any need to keep this backwards compatible, or do we assume that ppl always regenerate stuff and there's no need to parse stuff which doesn't have `hashes` ?

Reply:
It is backwards-compatible in the sense that this is a convenience conversion table so that client code can quickly lookup a function name given its hash. Older client code that won't be regenerated will simply have to do the translation themselves.